test/fixtures/api/issue-702.json

```diff
@@ -155,6 +155,48 @@
                                 }
                               },
                               "content": "+ Body\n\n        {}\n\n+ Schema \n\n        {}\n"
+                            },
+                            {
+                              "element": "asset",
+                              "meta": {
+                                "classes": {
+                                  "element": "array",
+                                  "content": [
+                                    {
+                                      "element": "string",
+                                      "content": "messageBody"
+                                    }
+                                  ]
+                                }
+                              },
+                              "attributes": {
+                                "contentType": {
+                                  "element": "string",
+                                  "content": "application/json"
+                                }
+                              },
+                              "content": "{}"
```

Comment:
Is this correct? According to how the parser has interpreted the body, it is `+ Body\n\n        {}\n\n+ Schema \n\n        {}\n` according to the asset above. We don't generate assets when the user have provided one.